data/remnant/remnant 2 side missions.txt

```diff
@@ -640,3 +640,59 @@ mission "Remnant: Shattered Light 6"
 			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
 			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
 			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`
+
+
+
+mission "Remnant: Scent of Rain 1"
+	name "Search for the unknown Korath ship."
+	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."
+	landing
+	source
+		government "Remnant"
+	destination "Caelian"
+	to offer
+		has "Remnant: Cognizance 25: done"
+	npc "scan outfits"
+		government "Korath"
+		personality waiting target
+		system
+			system "Kor Zena'i"
+		ship "Tubfalet" "Ra'puk At 295"
+		dialog `In poor lighting this ship might have passed for a Palavret, but major structural difference suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`
```

Comment:
```suggestion
		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a different model, or perhaps an entirely different ship. Time to report this to Chilia.`
```
`this is at least a different model, or perhaps an entirely different ship`
What does this mean? It seems like a tautology.

Reply:
What I was going for is conveying that the pilot isn't really sure what it is: Seen at a glance, this ship could be mistaken for a Palavret. But with the scans the structural detail makes it clear that this is at least a different version of the ship that we have seen to date, maybe even a completely new ship that just happens to have a strong surface resemblence to the Palavret.

Reply:
```suggestion
		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a modified design, or perhaps an entirely different ship. Time to report this to Chilia.`
```
`different model` and `entirely different ship` don't seem like meaningfully different terms. Alternatively:
```suggestion
		dialog `In poor lighting, this ship might have passed for a Palavret, but major structural differences suggest this is at least a new generation of the model, or perhaps an entirely different ship. Time to report this to Chilia.`
```